test/Database/Mongo/Update.purs

```diff
@@ -0,0 +1,32 @@
+module Test.Database.Mongo.Update where
+
+import Prelude (Unit, bind, (==), ($), (<<<))
+import Data.Maybe (Maybe(..))
+import Data.Tuple (Tuple)
+import Data.StrMap (fromFoldable)
+import Data.Argonaut as A
+import Data.Argonaut.Core (Json)
+import Data.Argonaut.Encode (encodeJson)
+import Control.Monad.Eff.Class (liftEff)
+import Control.Monad.Eff.Console (log)
+import Test.Data.Event (Event(..))
+import Test.Assert (assert)
+import Database.Mongo.Mongo (Database, updateOne, collection)
+import Database.Mongo.Bson.BsonValue as B
+import Database.Mongo.Options
+import Test.Type (Test)
+
+evt :: Event
+evt = Event
+  { name : Just "Wow"
+  }
+
+obj :: Array (Tuple String Json) -> Json
+obj = A.fromObject <<< fromFoldable
+
+main :: Database -> Test Unit
+main database = do
+  liftEff $ log "should update"
+  col <- collection "events" database
+  res <- updateOne ["name" B.:= "Wow"] ["name" B.:= "lol"] defaultUpdateOptions col
+  liftEff $ assert $ (obj [ "ok" A.:= 1, "n" A.:= 1, "nModified" A.:= 1, "nInserted" A.:= 0]) == (encodeJson res)
```

Comment:
I don't like this actually. 

`res` here is a `WriteResult`, to verify it, It's better to construct another `WriteResult` then compare. But I can't take this approach, because [the constructor of `WriteResult` is not exported.](https://github.com/zhangchiqing/purescript-node-mongodb/blob/master/src/Database/Mongo/Results.purs#L2)

That's why I used this workaround here which will encode the `WriteResult` into `Json`, then create another `Json` to compare. I don't like it because it's testing two functions together: `insertOne` + `encodeJson`. 

Why do you think? Is there better pattern for asserting the result?